index.ts

```diff
@@ -101,7 +129,7 @@ async function main() {
           // - any label
           if (
             devpoolIssue.title !== projectIssue.title ||
-            devpoolIssue.state !== projectIssue.state ||
+            (devpoolIssue.state !== projectIssue.state && !projectIssue.assignee) ||
```

Comment:
If project issue is solved/closed by some user (i.e. project's issue assignee exists) then such issue won't be synced with [devpool issues](https://github.com/ubiquity/devpool-directory/issues).

Shouldn't it we keep the old code?
```
devpoolIssue.state !== projectIssue.state ||
```

Reply:
that wouldnt work because if a open project issue has a assignee, it should be closed in the devpool but after closing the issue in the devpool, that check opened the issue.

Reply:
If project issue is closed with assignee and devpool issue is still open then [this](https://github.com/ubiquity/devpool-directory/blob/add9984a8ee82be14d14ea3171bf518c91bc07f2/index.ts#L130) line evaluates to `false` hence the devpool issue won't be closed [here](https://github.com/ubiquity/devpool-directory/blob/add9984a8ee82be14d14ea3171bf518c91bc07f2/index.ts#L136)